src/pages/Checklist/index.tsx

```diff
@@ -387,11 +387,11 @@ export const Checklist = () => {
       discord: 'https://discord.gg/z9efH7e',
       defaultTcp: 13000,
       defaultUdp: 12000,
-      jwtUrl: 'https://docs.prylabs.network/docs/execution-node/authentication',
+      jwtUrl: 'https://prysm.offchainlabs.com/docs/configure-prysm/authentication/',
       feeRecipientUrl:
-        'https://docs.prylabs.network/docs/execution-node/fee-recipient',
+        'https://prysm.offchainlabs.com/docs/configure-prysm/authentication/',
```

Comment:
this points to an unrelated page. it should be:
```suggestion
        'https://prysm.offchainlabs.com/docs/configure-prysm/fee-recipient/',
```

Reply:
Thank you! 👍